### DISCUSSION TITLE
Welcome to BDFramework Discussions!

### DISCUSSION THREAD

**Discussion by yimengfan** [yimengfan/BDFramework-Unity3d#51] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yimengfan** [comment]
[Test]我来组成头部!

**Comment by wqaetly** [comment]
嗨嗨嗨，来啦

**Reply by yimengfan** [reply]
nb

**Comment by Wanda0104** [comment]
嗨嗨嗨，来啦

**Comment by TuzkiG** [comment]
嗨嗨嗨  没有新群了吗

**Comment by li10400** [comment]
已收到，谢谢！

**Comment by TuzkiG** [comment]
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
